packages/jest-worker/__benchmarks__/test.js

```diff
@@ -5,22 +5,22 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+// Typical tests: node --expose-gc test.js empty 100000
+//                node --expose-gc test.js loadTest 10000
+
 'use strict';
 
 const assert = require('assert');
 const {performance} = require('perf_hooks');
-// eslint-disable-next-line import/no-extraneous-dependencies
 const workerFarm = require('worker-farm');
-const JestWorker = require('../../build').Worker;
+const JestWorker = require('../build').Worker;
```

Comment:
Not sure if it is good idea to reach into `build`. All of benchmark scripts are set in this way. Perhaps that's fine in this case?

Reply:
if `../` works I'd prefer that

Reply:
Yes, it works.